components/Player/PlayerName.jsx

```diff
@@ -1,31 +1,60 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { CardHeader } from 'material-ui/Card';
-// import PlayerPicture from './PlayerPicture';
+import AccountCircle from 'material-ui/svg-icons/action/account-circle';
+import { green500, grey600 } from 'material-ui/styles/colors';
+import FontIcon from 'material-ui/FontIcon';
 import Error from '../Error';
 import Spinner from '../Spinner';
 import { player } from '../../reducers';
-// import styles from './PlayerHeader.css';
+import styles from './PlayerHeader.css';
+import palette from '../palette.css';
+
+const PlayerName = ({ playerName, picture, cheese, registered, loading, error }) => {
```

Comment:
tracked_until is returned in the player API call.  Modify the reducer to store that state and the selector to read it, then use it directly here.